public/stylesheets/accessScoreDemo.css

```diff
@@ -51,6 +51,29 @@ svg {
     padding: 5px 0 0 0;
 }
 
+#reset-button {
```

Comment:
It looks like this CSS is identical to the CSS for the reset button in choropleth.css. Can you make it so that it is only located in one place?